src/offset/fixed.rs

```diff
@@ -128,6 +130,12 @@ impl Offset for FixedOffset {
     fn fix(&self) -> FixedOffset {
         *self
     }
+    #[cfg(feature = "std")]
+    fn now(&self) -> DateTime<FixedOffset> {
+        let now =
```

Comment:
How about writing this as `Utc::now().with_timezone(self)`?

Reply:
I started that way then eventually wondered if FixedOffset should have it's own now()

Reply:
oh. i understand. yes.